src/components/FormField/StatementReviewsTable/StatementReview.tsx

```diff
@@ -40,8 +41,8 @@ const StatementReview = ({
   } = value;
 
   const { data: author = createdBy } = useQuery(
-    ['/query', { target: [createdBy] }],
-    ({ queryKey: [route, body] }) => api.post(route, body),
+    tuple('/query', { target: [createdBy] }),
+    ({ queryKey: [_, body] }) => api.query(body),
```

Comment:
you use lodash, but you never import lodash as `_` so I can't see that causing confusion. I like to use the underscore convention because it's much clearer that you're intentionally ignoring an argument and didn't just accidentally type an extra comma

Reply:
I had stopped using it generally b/c it doesn't work for 2+ ignored arguments so you end up not being able to consistently. However I don't feel strongly about it so if you do feel free to leave it as is.

Reply:
I generally do either increasing numbers of underscores or _argname for multiple ignored args

Reply:
I grepped the codebase and did find some existing example of the pattern you suggested, so I've swapped to avoid competing conventions
[adce433](https://github.com/bcgsc/pori_graphkb_client/pull/52/commits/adce433e8cef1d2cb6456174ad63aeb745b4a67c)